core/pkg/ingress/controller/controller.go

```diff
@@ -650,26 +655,18 @@ func (ic *GenericController) getBackendServers() ([]*ingress.Backend, []*ingress
 						if location.Path == rootLocation {
 							if location.Backend == defUpstreamName {
 								glog.Warningf("ignoring ssl passthrough of %v as it doesn't have a default backend (root context)", server.Hostname)
-							} else {
-								isHTTPSfrom = append(isHTTPSfrom, server)
+								continue
 							}
+
+							isHTTPSfrom = append(isHTTPSfrom, server)
 						}
-					} else {
-						isHTTP = true
+						continue
 					}
 				}
 			}
 		}
 		if len(isHTTPSfrom) > 0 {
-			if isHTTP {
-				for _, server := range isHTTPSfrom {
-					glog.Warningf("backend type mismatch on %v, assuming HTTP on ssl passthrough host %v", upstream.Name, server.Hostname)
```

Comment:
This warning means the same backend is receiving http and https requests from the ingress controller. Any reason to remove it?